astroquery/heasarc/core.py

```diff
@@ -261,7 +259,7 @@ def query_mission_cols(self, mission, *, cache=True,
         cols = [col.upper() for col in cols['name'] if '__' not in col]
         return cols
 
-    def query_tap(self, query, *, maxrec=None):
+    def query_tap(self, query, *, maxrec=None, uploads=None):
```

Comment:
I would think propagating the table uploads could be super useful for the other methods calling `query_tap` internally, so please consider adding it to `query_region` and the new `query_by_column`, too.